np.sh

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env sh
+
+if test -n "$(git status --porcelain)"; then
```

Comment:
Hmm, I just realized this also will complain about untracked files. I think that could be an unwanted feature. Switching the command with `git status -uno --porcelain` would fix that.